### DISCUSSION TITLE
Is it possible to include @imput or @select event

### DISCUSSION THREAD

**Discussion by hugorodriguesbr** [elreco/vue-tailwind-datepicker#80] [General]
I would like to use the datepicker as a filter, emitting an event when selecting a date.

example:
<vue-tailwind-datepicker 
                i18n="pt-br"
                placeholder="Data"
                as-single use-range 
                separator=" Até "
                :formatter="dateFormat"
                :disable-date="dateDisable"
                :options="dateoptions"  
                v-model="dateValue"
                @imput="onSelectSomething"
                />

**Comment by xavier-ottolini** [comment]
I have got the same problem as you. I have found a workaround. Here is an example below:


```typescript
<script setup lang="ts">
const periodValue: Ref<{
        startDate: string, 
        endDate: string
    }> = ref({
                startDate: '2023-04-25',
                endDate: '2023-04-26'
            });
const onDateRangeChanged = () => {
    emit('update:modelValue', {
        startDate: periodValue.value.startDate,
        endDate: periodValue.value.endDate
    });
};

watch(() => periodValue.value.startDate, onDateRangeChanged)
watch(() => periodValue.value.endDate, onDateRangeChanged)
</script>
<template>
<vue-tailwind-datepicker v-model="periodValue" />
</template>
```
